staging/src/k8s.io/apimachinery/pkg/util/validation/field/errors.go

```diff
@@ -221,6 +222,9 @@ func InternalError(field *Path, err error) *Error {
 	return &Error{ErrorTypeInternal, field.String(), nil, err.Error()}
 }
 
+func ChangeForbidden(field *Path, value interface{}, detail string) *Error {
```

Comment:
add godoc describing the meaning and use, that this is *only* for errors where a field change is disallowed, and callers must also include errors if the new value is invalid independent of the current value